pkg/webhooks/validation/controller/controller.go

```diff
@@ -117,9 +117,11 @@ func newController(o Options, client kube.Client) *Controller {
 		controllers.WithReconciler(c.Reconcile),
 		// Webhook patching has to be retried forever. But the retries would be rate limited.
 		controllers.WithMaxAttempts(math.MaxInt),
-		// Try first few(5) retries quickly so that we can detect true conflicts by multiple Istiod instances fast.
-		// If there is a conflict beyond this, it means Istiods are seeing different ca certs and are in inconsistent
-		// state for longer duration. Slowdown the retries, so that we do not overload kube api server and etcd.
+		// Retry with backoff. Failures could be from conflicts of other instances (quick retry helps), or
+		// longer lasting concerns which will eventually be retried on 1min interval.
+		// Unlike the mutating webhook controller, we do not use NewItemFastSlowRateLimiter. This is because
```

Comment:
Unlike the mutating webhook controller, we do not use NewItemFastSlowRateLimiter - This is not clear to me

Reply:
In MWH controller, we use NewItemFastSlowRateLimiter. Therefor its intuitive to use the same here (which is why we used to). But its wrong (as noted in comments)

Reply:
You mean even though we configure here with `NewItemFastSlowRateLimiter` - we never use its "fast" part but always go in the slow path? If that is your intention -  it is not coming out clearly.

Reply:
Ahhh.. no I messed up and missed half the change :upside_down_face:... thank you! Was supposed to change this to exponentional backoff

Reply:
https://github.com/istio/istio/pull/44660